config/dispatcher/dispatcher.ex

```diff
@@ -1,134 +1,140 @@
+  match "/geslacht-codes/*path", @api do
+    forward conn, path, "http://cache/geslacht-codes/"
   end
-  match "/rechtsgronden-aanstelling/*path" do
-    Proxy.forward conn, path, "http://cache/rechtsgronden-aanstelling/"
+  match "/identificatoren/*path", @api do
+    forward conn, path, "http://cache/identificatoren/"
   end
-  match "/rechtsgronden-beeindiging/*path" do
-    Proxy.forward conn, path, "http://cache/rechtsgronden-beeindiging/"
+  match "/rechtsgronden-aanstelling/*path", @api do
+    forward conn, path, "http://cache/rechtsgronden-aanstelling/"
   end
-  match "/rechtstreekse-verkiezingen/*path" do
-    Proxy.forward conn, path, "http://cache/rechtstreekse-verkiezingen/"
+  match "/rechtsgronden-beeindiging/*path", @api do
+    forward conn, path, "http://cache/rechtsgronden-beeindiging/"
   end
-  match "/rechtsgronden/*path" do
-    Proxy.forward conn, path, "http://cache/rechtsgronden/"
+  match "/rechtstreekse-verkiezingen/*path", @api do
+    forward conn, path, "http://cache/rechtstreekse-verkiezingen/"
   end
-  match "/tijdsgebonden-entiteiten/*path" do
-    Proxy.forward conn, path, "http://cache/tijdsgebonden-entiteiten/"
+  match "/rechtsgronden/*path", @api do
+    forward conn, path, "http://cache/rechtsgronden/"
   end
-  match "/tijdsintervallen/*path" do
-    Proxy.forward conn, path, "http://cache/tijdsintervallen/"
+  match "/tijdsgebonden-entiteiten/*path", @api do
+    forward conn, path, "http://cache/tijdsgebonden-entiteiten/"
   end
-  match "/verkiezingsresultaten/*path" do
-    Proxy.forward conn, path, "http://cache/verkiezingsresultaten/"
+  match "/tijdsintervallen/*path", @api do
+    forward conn, path, "http://cache/tijdsintervallen/"
   end
-  match "/verkiezingsresultaat-gevolg-codes/*path" do
-    Proxy.forward conn, path, "http://cache/verkiezingsresultaat-gevolg-codes/"
+  match "/verkiezingsresultaten/*path", @api do
+    forward conn, path, "http://cache/verkiezingsresultaten/"
   end
-  match "/files/*path" do
-    Proxy.forward conn, path, "http://filehost/"
+  match "/verkiezingsresultaat-gevolg-codes/*path", @api do
+    forward conn, path, "http://cache/verkiezingsresultaat-gevolg-codes/"
   end
-  get "/exports/*path" do
+  get "/exports/*path", @api do
     # we bypass the cache on purpose since mu-cl-resources is not the master of the exports
-    Proxy.forward conn, path, "http://resource/exports/"
+    forward conn, path, "http://resource/exports/"
   end
-  get "/sitemap.xml" do
-    Proxy.forward conn, [], "http://sitemap/sitemap.xml"
+  get "/sitemap.xml", @api do
+    forward conn, [], "http://sitemap/sitemap.xml"
   end
 
   ###############################################################
   # delta-files
   ###############################################################
-  get "/delta-files/:id/download" do
-    Proxy.forward conn, [], "http://file/files/" <> id <> "/download"
+  get "/delta-files/:id/download", %{ layer: :api_services, accept: %{ any: true } } do
```

Comment:
Not sure if the accept is needed here. I _think_ it defaults to `*/*` if we don't set this?

Reply:
Not clear to from the doc, but the doc seems hinting at it: https://github.com/mu-semtech/mu-dispatcher?tab=readme-ov-file#host-an-emberjs-app